Framework/Algorithms/src/SofQWNormalisedPolygon.cpp

```diff
@@ -21,38 +24,259 @@
 #include "MantidKernel/PhysicalConstants.h"
 #include "MantidTypes/SpectrumDefinition.h"
 
+#include <boost/math/special_functions/pow.hpp>
+
 using Mantid::Geometry::rad2deg;
+using boost::math::pow;
```

Comment:
Do you have any specific reason for `boost::math::pow` and not `std::pow`?

Reply:
Mostly out of an old habit as both produce the same code when optimizations are turned on. However, in debug mode `boost::pow<>()` produces nicer assembly.